### ISSUE TITLE
Airloop water coils do not respond to EMS On/Off Supervisory control

### ISSUE THREAD

**Issue by rraustad** [NREL/EnergyPlus#5650]
## Issue overview

UH [question](https://unmethours.com/question/17142/energyplus-ems-manager-onoff-supervisory-for-heating-and-cooling-coils/) on water coil control using EMS. The air loop water coil does not respond to EMS component control using On/Off Supervisory control.
### Details

Some additional details for this issue (if relevant):
- Platform (Operating system, version)
- Version of EnergyPlus (if using an intermediate build, include SHA)
### Checklist

Add to this list or remove from it as applicable.  This is a simple templated set of guidelines.
- [x] Defect file added (list location of defect file here)
- [x] Ticket added to Pivotal for defect (development team task)
- [x] Pull request created (the pull request will have additional tasks related to reviewing changes that fix this defect)

**Comment by rraustad** [comment]
Simple addition of test for EMS control makes air loop coils respond to EMS command.
In SetActuatedBranchFlowRate function in PlantUtilities"

```
if ( comp.EMSLoadOverrideOn ) {
    if ( comp.EMSLoadOverrideValue == 0.0 ) {
        Node( NodeNum ).MassFlowRate = 0.0;
        Node( NodeNum ).MassFlowRateRequest = 0.0;
    }
}
```

**Comment by rraustad** [comment]
See PR https://github.com/NREL/EnergyPlus/pull/5651

**Comment by rraustad** [comment]
@EnergyArchmage @Myoldmopar This issue is closed via #5651.
